modules/core/nuxeo-core-el/src/test/java/org/nuxeo/platform/el/TestExpressionEvaluator.java

```diff
@@ -106,4 +107,21 @@ public void testList() {
         assertNotNull(value);
     }
 
+    /**
+     * NXP-28918
+     */
+    @Test
+    public void testArrayContainsFunction() {
```

Comment:
Please also test the `null` case.

Reply:
WDYM? The array to `null` or the element?

Reply:
I meant the array.

Reply:
Ah yes sorry